### ISSUE TITLE
ROS2 plans ?

### ISSUE THREAD

**Issue by doisyg** [carzum/termviz#77]
That would be awesome !

**Comment by carzum** [comment]
Yes! We already talked about that being a good next step. Ideally one would only change the subscribers and support ROS1 and 2 via feature flags.

**Comment by francocipollone** [comment]
> Yes! We already talked about that being a good next step. Ideally one would only change the subscribers and support ROS1 and 2 via feature flags.

That's great! Are you pushing this effort in a separate repository?

**Comment by james-yoo** [comment]
Any update?

**Comment by g-gemignani** [comment]
During the break I started the support of ros2. Still has to be fully tested https://github.com/g-gemignani/termviz/tree/feature/ros2-support

Talked with @carzum and he didn't hate the idea.

**Comment by g-gemignani** [comment]
If someone wants to give it a try, I just opened a MR supporting ros2 https://github.com/carzum/termviz/pull/102

**Comment by g-gemignani** [comment]
We just merged the ros2 version into main if someone wants to try it out. Closing this
